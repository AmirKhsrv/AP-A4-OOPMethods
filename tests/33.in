best_book
show_author_info 658
credit 23
recommend_second_approach 74
credit 44
show_author_info 3
show_author_info 1309
recommend_second_approach 27
best_reviewer
recommend_first_approach 62
recommend_first_approach 47
