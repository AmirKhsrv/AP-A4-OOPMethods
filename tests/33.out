id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 658
Name: Sona Charaipotra
Year of Birth: 1979
Place of Birth: Kuwait
Member Since: February 1 2008
Genres: Young Adult
Books:
0.006425
id: 177
Title: Disasterology 101
Genre: Romance
Author: Taylor V. Donovan
Reviews:
id: 973 Rating: 1 Likes: 0 Date: September 10 2019
id: 1143 Rating: 2 Likes: 2 Date: March 27 2015
id: 1343 Rating: 0 Likes: 0 Date: July 6 2013
id: 1444 Rating: 2 Likes: 0 Date: February 6 2016
id: 1753 Rating: 3 Likes: 0 Date: June 2 2016
id: 2235 Rating: 5 Likes: 1 Date: November 27 2015
id: 2375 Rating: 2 Likes: 0 Date: May 1 2019
id: 3060 Rating: 5 Likes: 0 Date: September 3 2018
id: 3925 Rating: 5 Likes: 0 Date: June 15 2012
id: 4134 Rating: 5 Likes: 0 Date: August 17 2017
id: 5279 Rating: 5 Likes: 0 Date: January 14 2016
id: 5280 Rating: 2 Likes: 0 Date: May 10 2016
id: 6593 Rating: 3 Likes: 9 Date: February 13 2017
id: 6981 Rating: 2 Likes: 2 Date: June 29 2017
id: 8636 Rating: 4 Likes: 0 Date: February 13 2019
id: 8688 Rating: 1 Likes: 0 Date: May 10 2012
id: 9550 Rating: 3 Likes: 0 Date: October 1 2018
id: 10543 Rating: 0 Likes: 0 Date: August 3 2010
id: 10658 Rating: 4 Likes: 5 Date: February 7 2015
id: 10661 Rating: 1 Likes: 5 Date: November 7 2019
id: 10964 Rating: 2 Likes: 90 Date: March 23 2016
id: 11483 Rating: 1 Likes: 11 Date: January 5 2019
id: 11667 Rating: 4 Likes: 0 Date: May 17 2019
id: 12054 Rating: 3 Likes: 3 Date: February 29 2011
id: 13276 Rating: 3 Likes: 0 Date: December 17 2017
id: 15316 Rating: 4 Likes: 0 Date: August 23 2015
id: 15368 Rating: 3 Likes: 0 Date: June 4 2019
id: 15515 Rating: 4 Likes: 0 Date: February 14 2019
id: 16433 Rating: 0 Likes: 0 Date: October 28 2018
id: 16793 Rating: 5 Likes: 0 Date: January 24 2015
id: 16830 Rating: 3 Likes: 0 Date: February 13 2017
0.005808
id: 3
Name: Tymber Dalton
Year of Birth: 1994
Place of Birth: Thailand
Member Since: July 9 2007
Genres: Romance
Books:
id: 1309
Name: R.C. Stephens
Year of Birth: 1995
Place of Birth: Pakistan
Member Since: September 20 2018
Genres: Romance
Books:
id: 303
Title: The Good Luck Charm
Genre: Romance
Author: Helena Hunting
Reviews:
id: 99 Rating: 3 Likes: 8 Date: November 16 2018
id: 142 Rating: 4 Likes: 0 Date: January 20 2019
id: 949 Rating: 1 Likes: 1 Date: May 14 2017
id: 1209 Rating: 1 Likes: 0 Date: August 22 2013
id: 1350 Rating: 5 Likes: 2 Date: September 25 2018
id: 1595 Rating: 1 Likes: 0 Date: July 1 2013
id: 1747 Rating: 1 Likes: 0 Date: November 13 2018
id: 1846 Rating: 4 Likes: 0 Date: July 9 2016
id: 3401 Rating: 3 Likes: 0 Date: January 1 2016
id: 3889 Rating: 5 Likes: 0 Date: October 17 2017
id: 4887 Rating: 5 Likes: 3 Date: May 4 2018
id: 5127 Rating: 5 Likes: 0 Date: November 9 2010
id: 5574 Rating: 3 Likes: 0 Date: October 6 2014
id: 6137 Rating: 3 Likes: 0 Date: June 28 2017
id: 6480 Rating: 1 Likes: 1 Date: May 1 2011
id: 7559 Rating: 0 Likes: 7 Date: August 19 2017
id: 7913 Rating: 3 Likes: 0 Date: July 10 2019
id: 8048 Rating: 2 Likes: 0 Date: December 23 2018
id: 8160 Rating: 1 Likes: 0 Date: February 26 2017
id: 8310 Rating: 2 Likes: 0 Date: November 29 2019
id: 8626 Rating: 5 Likes: 0 Date: June 16 2014
id: 8683 Rating: 4 Likes: 0 Date: June 20 2013
id: 8858 Rating: 0 Likes: 0 Date: December 16 2015
id: 8862 Rating: 4 Likes: 0 Date: August 26 2018
id: 8903 Rating: 5 Likes: 0 Date: July 11 2018
id: 9182 Rating: 2 Likes: 0 Date: October 12 2018
id: 10427 Rating: 2 Likes: 0 Date: July 28 2018
id: 10432 Rating: 3 Likes: 0 Date: January 9 2018
id: 10490 Rating: 4 Likes: 0 Date: October 20 2017
id: 11047 Rating: 2 Likes: 24 Date: October 15 2010
id: 12102 Rating: 2 Likes: 3 Date: April 22 2018
id: 12147 Rating: 4 Likes: 2 Date: December 12 2019
id: 12633 Rating: 3 Likes: 0 Date: March 21 2016
id: 13302 Rating: 4 Likes: 0 Date: April 18 2015
id: 14111 Rating: 3 Likes: 1 Date: September 14 2016
id: 14698 Rating: 2 Likes: 0 Date: October 22 2019
id: 16634 Rating: 2 Likes: 0 Date: October 13 2018
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 392
Title: From the Mixed-Up Files of Mrs. Basil E. Frankweiler
Genre: Fiction
Author: E.L. Konigsburg
Reviews:
id: 356 Rating: 3 Likes: 0 Date: August 24 2018
id: 396 Rating: 2 Likes: 558 Date: January 16 2011
id: 677 Rating: 5 Likes: 3 Date: August 13 2018
id: 914 Rating: 5 Likes: 0 Date: June 3 2019
id: 1654 Rating: 0 Likes: 0 Date: February 6 2019
id: 1665 Rating: 5 Likes: 0 Date: November 21 2017
id: 1866 Rating: 0 Likes: 0 Date: December 27 2018
id: 2173 Rating: 2 Likes: 0 Date: February 17 2012
id: 2285 Rating: 5 Likes: 0 Date: March 28 2013
id: 2901 Rating: 5 Likes: 0 Date: July 20 2017
id: 4481 Rating: 5 Likes: 2 Date: May 21 2016
id: 5827 Rating: 4 Likes: 0 Date: November 25 2019
id: 5975 Rating: 1 Likes: 0 Date: March 28 2018
id: 6899 Rating: 5 Likes: 1 Date: February 27 2018
id: 8914 Rating: 1 Likes: 0 Date: July 12 2013
id: 9248 Rating: 5 Likes: 0 Date: October 4 2018
id: 9999 Rating: 5 Likes: 0 Date: March 18 2015
id: 10317 Rating: 4 Likes: 0 Date: April 18 2014
id: 10521 Rating: 4 Likes: 0 Date: September 13 2019
id: 10822 Rating: 5 Likes: 0 Date: May 11 2016
id: 11689 Rating: 5 Likes: 1 Date: January 27 2017
id: 12274 Rating: 5 Likes: 0 Date: August 25 2015
id: 12379 Rating: 5 Likes: 0 Date: June 23 2015
id: 12431 Rating: 0 Likes: 1 Date: July 25 2019
id: 12626 Rating: 3 Likes: 0 Date: September 18 2019
id: 13444 Rating: 0 Likes: 0 Date: June 26 2013
id: 13813 Rating: 3 Likes: 0 Date: May 27 2018
id: 14083 Rating: 3 Likes: 0 Date: October 26 2014
id: 14142 Rating: 4 Likes: 0 Date: December 27 2014
id: 14199 Rating: 3 Likes: 4 Date: October 12 2015
id: 14427 Rating: 1 Likes: 2 Date: June 15 2018
id: 16431 Rating: 5 Likes: 0 Date: October 28 2014
id: 16509 Rating: 5 Likes: 0 Date: March 17 2016
id: 16740 Rating: 3 Likes: 0 Date: April 26 2016
id: 16829 Rating: 0 Likes: 0 Date: October 15 2017
id: 17340 Rating: 3 Likes: 0 Date: November 13 2017
id: 236
Title: Slow Burn
Genre: Romance
Author: Sam B. Morgan
Reviews:
id: 22 Rating: 0 Likes: 38 Date: February 27 2016
id: 709 Rating: 4 Likes: 2 Date: September 27 2013
id: 816 Rating: 2 Likes: 0 Date: November 29 2019
id: 1315 Rating: 5 Likes: 0 Date: October 16 2015
id: 1626 Rating: 2 Likes: 0 Date: March 28 2019
id: 1698 Rating: 3 Likes: 0 Date: July 2 2016
id: 2240 Rating: 2 Likes: 0 Date: March 4 2013
id: 2338 Rating: 3 Likes: 0 Date: March 29 2019
id: 2651 Rating: 2 Likes: 0 Date: November 25 2018
id: 2702 Rating: 5 Likes: 0 Date: October 5 2018
id: 2861 Rating: 3 Likes: 0 Date: April 11 2014
id: 4305 Rating: 5 Likes: 20 Date: May 8 2015
id: 4625 Rating: 0 Likes: 1 Date: March 8 2011
id: 5276 Rating: 5 Likes: 0 Date: September 4 2019
id: 6661 Rating: 4 Likes: 2 Date: January 11 2019
id: 6698 Rating: 3 Likes: 2 Date: December 29 2010
id: 6754 Rating: 5 Likes: 5 Date: July 23 2014
id: 7207 Rating: 4 Likes: 13 Date: June 27 2018
id: 7241 Rating: 3 Likes: 12 Date: October 20 2011
id: 7335 Rating: 1 Likes: 1 Date: April 15 2012
id: 7395 Rating: 2 Likes: 0 Date: September 9 2019
id: 7785 Rating: 2 Likes: 0 Date: January 27 2019
id: 7998 Rating: 4 Likes: 0 Date: March 19 2016
id: 8669 Rating: 3 Likes: 0 Date: May 2 2014
id: 9719 Rating: 0 Likes: 0 Date: November 24 2019
id: 9874 Rating: 2 Likes: 0 Date: August 25 2014
id: 10004 Rating: 4 Likes: 0 Date: March 25 2019
id: 10171 Rating: 4 Likes: 2 Date: February 14 2019
id: 10644 Rating: 2 Likes: 0 Date: June 25 2019
id: 10714 Rating: 4 Likes: 20 Date: June 6 2019
id: 10841 Rating: 3 Likes: 0 Date: April 27 2017
id: 10909 Rating: 5 Likes: 0 Date: April 9 2013
id: 11757 Rating: 0 Likes: 0 Date: March 27 2018
id: 12179 Rating: 5 Likes: 0 Date: September 5 2015
id: 13019 Rating: 5 Likes: 1 Date: May 21 2012
id: 13321 Rating: 4 Likes: 0 Date: May 18 2016
id: 14914 Rating: 1 Likes: 0 Date: November 17 2018
id: 16115 Rating: 5 Likes: 0 Date: April 7 2016
